rnn/species.cxx

```diff
@@ -87,71 +87,45 @@ void Species::copy_two_random_genomes(uniform_real_distribution<double> &rng_0_1
     *genome2 = genomes[p2]->copy();
 }
 
-
 //returns -1 for not inserted, otherwise the index it was inserted at
 //inserts a copy of the genome, caller of the function will need to delete their
 //pointer
 int32_t Species::insert_genome(RNN_Genome *genome) {
-
-    Log::debug("getting fitness of genome copy\n");
-    double new_fitness = genome->get_fitness();
     Log::info("inserting genome with fitness: %s to species %d\n", parse_fitness(genome->get_fitness()).c_str(), id);
 
-    // int32_t duplicate_genome_index = contains(genome);
```

Comment:
@travisdesell cleaning up some of the species implementation. Spoke with @zimenglyu